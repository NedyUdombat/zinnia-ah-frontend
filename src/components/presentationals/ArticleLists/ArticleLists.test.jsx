import React from 'react';
import { mount } from 'enzyme';
import { BrowserRouter } from 'react-router-dom';

import ArticleLists from './ArticleLists';

const props = {
  deleteArticle: jest.fn(),
  isDeleting: false,
  articles: [{}, {}],
};

describe('ArticleLists', () => {
  it('should render correctly', () => {
    const component = mount(
      <BrowserRouter>
        <ArticleLists {...props} />
      </BrowserRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
