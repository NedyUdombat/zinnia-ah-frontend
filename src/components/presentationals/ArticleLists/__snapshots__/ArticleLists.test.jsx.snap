// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleLists should render correctly 1`] = `
<ArticleLists
  articles={
    Array [
      Object {},
      Object {},
    ]
  }
  deleteArticle={[MockFunction]}
  isDeleting={false}
>
  <div
    className="post-lists"
  >
    <ItemGroup>
      <div
        className="ui items"
      >
        <Item>
          <div
            className="item"
          >
            <ItemImage
              src="https://react.semantic-ui.com/images/wireframe/image.png"
            >
              <Image
                as="img"
                src="https://react.semantic-ui.com/images/wireframe/image.png"
                ui={false}
                wrapped={true}
              >
                <div
                  className="image"
                >
                  <img
                    src="https://react.semantic-ui.com/images/wireframe/image.png"
                  />
                </div>
              </Image>
            </ItemImage>
            <ItemContent>
              <div
                className="content"
              >
                <div
                  className="d-flex"
                >
                  <div
                    style={
                      Object {
                        "marginRight": "auto",
                      }
                    }
                  >
                    <Title
                      className="title-article-lg"
                    >
                      <h1
                        className="title title-article-lg"
                      />
                    </Title>
                  </div>
                  <div
                    style={
                      Object {
                        "marginRight": "0.8rem",
                      }
                    }
                  >
                    <p
                      style={
                        Object {
                          "fontSize": "1rem",
                          "width": "100%",
                        }
                      }
                    >
                       mins read
                    </p>
                  </div>
                  <div>
                    <Button
                      className="btn-transparent"
                      type="button"
                      value=""
                    >
                      <button
                        className="btn btn-transparent"
                        type="button"
                      >
                        <Icon
                          as="i"
                          name="bookmark outline"
                          style={
                            Object {
                              "color": "#000000",
                              "fontSize": "1.25rem",
                              "margin": "0",
                            }
                          }
                        >
                          <i
                            aria-hidden="true"
                            className="bookmark outline icon"
                            onClick={[Function]}
                            style={
                              Object {
                                "color": "#000000",
                                "fontSize": "1.25rem",
                                "margin": "0",
                              }
                            }
                          />
                        </Icon>
                      </button>
                    </Button>
                  </div>
                </div>
                <ItemDescription
                  style={
                    Object {
                      "marginBottom": "auto",
                    }
                  }
                >
                  <div
                    className="description"
                    style={
                      Object {
                        "marginBottom": "auto",
                      }
                    }
                  >
                    <p
                      style={
                        Object {
                          "fontSize": "1.125rem",
                          "fontWeight": "200",
                          "lineHeight": "1.375rem",
                        }
                      }
                    />
                  </div>
                </ItemDescription>
                <ItemExtra>
                  <div
                    className="extra"
                  >
                    <div
                      className="d-flex justify-content-between"
                    >
                      <p
                        style={
                          Object {
                            "fontSize": "1rem",
                          }
                        }
                      >
                        a few seconds
                         ago
                      </p>
                      <div />
                    </div>
                  </div>
                </ItemExtra>
              </div>
            </ItemContent>
          </div>
        </Item>
      </div>
    </ItemGroup>
    <ItemGroup>
      <div
        className="ui items"
      >
        <Item>
          <div
            className="item"
          >
            <ItemImage
              src="https://react.semantic-ui.com/images/wireframe/image.png"
            >
              <Image
                as="img"
                src="https://react.semantic-ui.com/images/wireframe/image.png"
                ui={false}
                wrapped={true}
              >
                <div
                  className="image"
                >
                  <img
                    src="https://react.semantic-ui.com/images/wireframe/image.png"
                  />
                </div>
              </Image>
            </ItemImage>
            <ItemContent>
              <div
                className="content"
              >
                <div
                  className="d-flex"
                >
                  <div
                    style={
                      Object {
                        "marginRight": "auto",
                      }
                    }
                  >
                    <Title
                      className="title-article-lg"
                    >
                      <h1
                        className="title title-article-lg"
                      />
                    </Title>
                  </div>
                  <div
                    style={
                      Object {
                        "marginRight": "0.8rem",
                      }
                    }
                  >
                    <p
                      style={
                        Object {
                          "fontSize": "1rem",
                          "width": "100%",
                        }
                      }
                    >
                       mins read
                    </p>
                  </div>
                  <div>
                    <Button
                      className="btn-transparent"
                      type="button"
                      value=""
                    >
                      <button
                        className="btn btn-transparent"
                        type="button"
                      >
                        <Icon
                          as="i"
                          name="bookmark outline"
                          style={
                            Object {
                              "color": "#000000",
                              "fontSize": "1.25rem",
                              "margin": "0",
                            }
                          }
                        >
                          <i
                            aria-hidden="true"
                            className="bookmark outline icon"
                            onClick={[Function]}
                            style={
                              Object {
                                "color": "#000000",
                                "fontSize": "1.25rem",
                                "margin": "0",
                              }
                            }
                          />
                        </Icon>
                      </button>
                    </Button>
                  </div>
                </div>
                <ItemDescription
                  style={
                    Object {
                      "marginBottom": "auto",
                    }
                  }
                >
                  <div
                    className="description"
                    style={
                      Object {
                        "marginBottom": "auto",
                      }
                    }
                  >
                    <p
                      style={
                        Object {
                          "fontSize": "1.125rem",
                          "fontWeight": "200",
                          "lineHeight": "1.375rem",
                        }
                      }
                    />
                  </div>
                </ItemDescription>
                <ItemExtra>
                  <div
                    className="extra"
                  >
                    <div
                      className="d-flex justify-content-between"
                    >
                      <p
                        style={
                          Object {
                            "fontSize": "1rem",
                          }
                        }
                      >
                        a few seconds
                         ago
                      </p>
                      <div />
                    </div>
                  </div>
                </ItemExtra>
              </div>
            </ItemContent>
          </div>
        </Item>
      </div>
    </ItemGroup>
  </div>
</ArticleLists>
`;
